SubmitHGCalPGun.py

```diff
@@ -197,10 +202,12 @@ def submitHGCalProduction():
     partGunType = 'FlatRandom%sGunProducer' % opt.gunType
     if opt.gunMode == 'pythia8':
         partGunType = 'Pythia8%sGun' % opt.gunType
+    if 'physproc' in opt.gunMode:
```

Comment:
Just out of curiosity, can gunMode can be sth like `*physproc*`?

Reply:
Not anymore since I changed that. Will change to `==`